UInnovateApp/src/components/AdditionalViewNavBar.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react'
+import { Container, Nav, Navbar } from 'react-bootstrap'
+import { AdditionalViews, getCustomViews,  getViewsBySchema } from '../virtualmodel/AdditionalViewsDataAccessor';
+import { Table } from "../virtualmodel/VMD";
+import { ViewTypeEnum } from './settingsPage/additionalView/ViewTypeEnum';
+import e from 'express';
+
+interface AdditionalViewNavBarProp{
+    selectedSchema: string;
+    selectedTable?: Table;
+}
+const AdditionalViewNavBar = ({selectedSchema, selectedTable}: AdditionalViewNavBarProp) => {
+    const [viewList, setViewList] = useState<AdditionalViews[]>([]);
+    const [customViews, setCustomViews]= useState([]);
+
+    useEffect(()=>{
+        const ctrl = new AbortController();
+        const signal = ctrl.signal;
+        // get data from db
+        getViewsBySchema(setViewList, selectedSchema, signal);
+        // getCustomViews(setCustomViews, signal)

```

Comment:
Do we need this?

Reply:
yes that is actually the requirement for #230.